youtubeExtension/options.js

```diff
@@ -0,0 +1,86 @@
+function saveOptions() {
+
+  const checkboxIds = ['Profanity', 'Violence', 'Politics', 'Substances', 'Horrer'];
+  let options = {};
+
+  // loop through each checkbox id and save its state
+  checkboxIds.forEach(id => {
+    options[id] = document.getElementById(id).checked;
+  });
+
+  chrome.storage.sync.set(options);
+
+}
+
+function saveRange(){
+  const range = document.getElementById("my-range");
+  chrome.storage.sync.set({'my-range': slider.value});
+  console.log({'my-range': slider.value});
+}
+
+function restoreOptions() {
+
+  const checkboxIds = ['Profanity', 'Violence', 'Politics', 'Substances', 'Horrer'];
```

Comment:
You have these options defined in multiple places. I'd pull it out into a global constant, or make a function called `getCheckboxIds`. The constant approach is more common, but since you might let the user define these toggle options, it might make more sense to make a function (is it really a global constant if it can change?)